10-reading-data-into-R.Rmd

```diff
@@ -0,0 +1,275 @@
+# Loading survey data into R
+
+<!--TODO: add intro-->
+
+<!-- consider making this section a call out-->
+## Project-based workflows
+
+We recommend a project-based workflow for analysis projects as described in Hadley Wickham and Garrett Grolemund's book, R for Data Science, found at [r4ds.had.co.nz](https://r4ds.had.co.nz). A project-based workflow helps us practice file system discipline, since we put everything related to a project in a designated folder. Since all associated files are in a single location, they are easy to find and organize.
```

Comment:
```suggestion
We recommend a project-based workflow for analysis projects as described in Hadley Wickham and Garrett Grolemund's book, R for Data Science, found at [r4ds.had.co.nz](https://r4ds.had.co.nz). A project-based workflow helps us practice file system discipline, since we put everything related to a project in a designated folder. Because all associated files are in a single location, they are easy to find and organize.
```

Reply:
I removed "Chapter 10" from this branch and will create a new branch for it.